extensions/amp-story-interactive/0.1/amp-story-interactive-results-detailed.js

```diff
@@ -14,19 +14,57 @@
  * limitations under the License.
  */
 
+import {InteractiveType} from './amp-story-interactive-abstract';
 import {
-  AmpStoryInteractive,
-  InteractiveType,
-} from './amp-story-interactive-abstract';
+  AmpStoryInteractiveResults,
+  decideStrategy,
+} from './amp-story-interactive-results';
 import {CSS} from '../../../build/amp-story-interactive-results-detailed-0.1.css';
 import {htmlFor} from '#core/dom/static-template';
+import {setImportantStyles} from '#core/dom/style';
 
-export class AmpStoryInteractiveResultsDetailed extends AmpStoryInteractive {
+/**
+ * @typedef {{
+ *    el: !Element,
+ *    answered: boolean
+ * }} ResultEl
```

Comment:
Type should be named "ResultElementType" (not how types end with "Type"), and the property should be "element" (it gets optimized with JS in the minified bundle, so the length of the property name is not important)